python/docs/source/getting_started/install.rst

```diff
@@ -155,12 +155,11 @@ Dependencies
 Package       Minimum supported version Note
 ============= ========================= ======================================
 `pandas`      1.0.5                     Optional for Spark SQL
-`NumPy`       1.7                       Required for MLlib DataFrame-based API
 `pyarrow`     1.0.0                     Optional for Spark SQL
-`Py4J`        0.10.9.5                  Required
+`py4j`        0.10.9.5                  Required
 `pandas`      1.0.5                     Required for pandas API on Spark
 `pyarrow`     1.0.0                     Required for pandas API on Spark
-`Numpy`       1.14                      Required for pandas API on Spark
+`numpy`       1.15                      Required for pandas API on Spark and MLLib DataFrame-based API
```

Comment:
Bump from 1.14 to 1.15.

Reply:
https://github.com/apache/spark/blob/8765eea1c08bc58a0cfc22b7cfbc0b5645cc81f9/python/setup.py#L270-L274
https://github.com/apache/spark/blob/8765eea1c08bc58a0cfc22b7cfbc0b5645cc81f9/python/setup.py#L264-L265